flink-end-to-end-tests/flink-end-to-end-tests-pulsar/pom.xml

```
@@ -31,8 +31,8 @@ under the License.
 	<name>Flink : E2E Tests : Pulsar</name>
 
 	<properties>
-		<pulsar.version>2.10.0</pulsar.version>
-		<bouncycastle.version>1.69</bouncycastle.version>
+		<pulsar.version>2.10.1</pulsar.version>
+		<bouncycastle.version>1.70</bouncycastle.version>
```

```suggestion
		<bouncycastle.version>1.69</bouncycastle.version>
```

We should keep the same version with pulsar-client.